### DISCUSSION TITLE
How to plot histogram of properties?

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [PMEAL/OpenPNM#1810] [Q&A] [answered]

**Comment by ma-sadeghi** [comment, marked as answer]
You can do

```python
object.show_hist(propnames)
```

where `object` is an OpenPNM object and `propnames` is a string or a list of strings (propnames for which you need histogram).

**Reply by jgostick** [reply]
This is a little known feature but is quite handy.  As of version 2.6 is has *much nicer* default formatting.
